desktop/libs/aws/src/aws/conf.py

```diff
@@ -91,6 +92,28 @@ def get_default_region():
         key='region',
         default='us-east-1',
         type=str
+      ),
+      PROXY_ADDRESS=Config(
+        help='Alternate address for endpoint.',
```

Comment:
Could you use _() on all the help string to internationalize them? 
(and just noticed we forgot to do this in one above :)